.circleci/config.yml

```diff
@@ -60,36 +64,36 @@ default-context: &default-context
 commands:
   install_js:
     parameters:
-      browsers:
+      playwright:
+        description: If playwright should be installed.
         type: boolean
         default: false
-        description: 'Set to true if you intend to any browser (for example with playwright).'
       react-version:
+        description: The version of React to use.
         type: string
         default: << pipeline.parameters.react-version >>
-        description: The version of React to use.
+      node-version:
+        description: The version of node to be used
+        type: string
+        default: << pipeline.parameters.node-version >>
 
     steps:
-      - when:
-          condition: << parameters.browsers >>
-          steps:
-            - run:
-                name: Install pnpm package manager
-                command: corepack enable
-      - when:
-          condition:
-            not: << parameters.browsers >>
-          steps:
-            - run:
-                name: Install pnpm package manager
-                # See https://stackoverflow.com/a/73411601
-                command: corepack enable --install-directory ~/bin
+      - run:
+          name: Install pnpm package manager
+          # See https://stackoverflow.com/a/73411601
+          command: corepack enable --install-directory ~/bin
 
       - run:
           name: View install environment
           command: |
             node --version
             pnpm --version
+
+      - restore_cache:
+          name: Restore pnpm cache
+          keys:
+            - pnpm-cache-{{ checksum "pnpm-lock.yaml" }}
```

Comment:
The usefulness of pnpm store caching is debatable.
Maybe we could keep it for a standalone PR to assess its usefulness in the current state of our dependencies? 🤔